app/models/new_story_repo.rb

```diff
@@ -0,0 +1,25 @@
+class NewStoryRepo
+  def initialize( client = Firebase::Client.new('https://hacker-news.firebaseio.com/v0') )
+    @client =  client
+  end
+
+  def story_with(story_id)
+    present story(story_id)
+  end
+
+  private
+  def story(story_id)
+    @client.get("item/#{story_id}", print: 'pretty').body
+  end
+
+  def present(story)
```

Comment:
Usually this kind of logic would be in a presenter/decorator class, but I guess we're not really decorating for the view, so it's ok here